server/src/main/java/com/cloud/network/vpc/VpcManagerImpl.java

```diff
@@ -1713,9 +1713,9 @@ public void validateNtwkOffForVpc(final NetworkOffering guestNtwkOff, final List
          * ("No redunant router support when network belnogs to VPC"); }
          */
 
-        // 4) Conserve mode should be off
+        // 4) Conserve mode should be off in elder versions
         if (guestNtwkOff.isConserveMode()) {
-            throw new InvalidParameterValueException("Only networks with conserve mode Off can belong to VPC");
+            s_logger.info("Creating a network with conserve mode in VPC");
         }
```

Comment:
can´t we just delete? (is this info useful?)

Reply:
+1